lib/capistrano/gitflow/helpers/helper.rb

```diff
@@ -216,5 +216,26 @@ def gitflow_tag_production
       set :branch, new_production_tag
     end
 
+    def gitflow_cleanup_tags
+      return if fetch(:gitflow_keep_tags).nil?
+      tags = `git log --tags  --pretty="format:%at %D" | grep 'tag:' |sort -n | awk '{$1=""; print $0}' | tr "," "\n"| sed 's/tag:*//' | sed -e 's/^[ \t]*//'`
+      tags = tags.split.reject{|tag| tag.blank?  }
```

Comment:
Fixed in this commit: https://github.com/technicalpickles/capistrano-gitflow/commit/df73e4fecb0544a9290ef89c80ae5e8551362d45 

And released version 1.5.7.